django/db/models/query_utils.py

```diff
@@ -90,22 +90,26 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
         clause, _ = query._add_q(self, reuse, allow_joins=allow_joins)
         return clause
 
-    def refs_aggregate(self, existing_aggregates):
-        def _refs_aggregate(obj, existing_aggregates):
-            if not isinstance(obj, tree.Node):
-                aggregate, aggregate_lookups = refs_aggregate(obj[0].split(LOOKUP_SEP), existing_aggregates)
-                if not aggregate and hasattr(obj[1], 'refs_aggregate'):
-                    return obj[1].refs_aggregate(existing_aggregates)
+    @classmethod
+    def _refs_aggregate(cls, obj, existing_aggregates):
+        if not isinstance(obj, tree.Node):
+            aggregate, aggregate_lookups = refs_aggregate(obj[0].split(LOOKUP_SEP), existing_aggregates)
+            if not aggregate:
+                obj_refs_aggregate = getattr(obj[1], 'refs_aggregate', None)
```

Comment:
I can't really think of anything :( maybe an extra ``_`` is enough to make the meaning clearer: ``obj__refs_aggregate``?